README.md

```diff
@@ -8,40 +8,44 @@
+
+If you are a member of the GOV.UK Account team, as a colleague if they have existing config they can share with you.
+
+If you need to generate new config follow the steps below.
 
 1. [Generate a key pair](https://auth-tech-docs.london.cloudapps.digital/integrate-with-integration-environment/generate-a-key/)
-1. Copy an existing client in the database, or [register a new one](https://auth-tech-docs.london.cloudapps.digital/integrate-with-integration-environment/manage-your-service-s-configuration/#manage-your-service-s-configuration-with-gov-uk-sign-in).
-1. In the database update the following values:
-    - SubjectType = public
-    - ConsentRequired = 0
-    - RedirectUrls = ["http://localhost:6001/auth/callback"]
-    - Scopes = ["openid", "phone", "email", "am", "offline_access", "govuk-account"]
-1. Copy and paste the public key from step 1.
+1. Ask the [Auth team](https://di-team-manual.london.cloudapps.digital/authentication/) to copy an existing client in the database, or [register a new one](https://auth-tech-docs.london.cloudapps.digital/integrate-with-integration-environment/manage-your-service-s-configuration/#manage-your-service-s-configuration-with-gov-uk-sign-in).
+1. If the Auth team ask for configuration values, tell them we need:
+   - SubjectType = public
+   - ConsentRequired = 0
+   - RedirectUrls = ["http://localhost:6001/auth/callback"]
+   - Scopes = ["openid", "phone", "email", "am", "offline_access", "govuk-account"]
+1. Send the Auth team the public key generated in step 1.
 
 ### Set the Environment variables
 
-Create a copy of the .env.sample file, rename it .env and fill in the value for the client id below.  All the other values should be correct.
+Create a copy of the .env.sample file, rename it .env and fill in the value for the client id below. All the other values should be correct.
 
 ```
 OIDC_CLIENT_ID=<client id>
 ```
 
 ### Setup the private key
 
-Copy the private key generated in step one and put it in `seed.yaml` with the correct identation.
+Create a copy of the seed.yaml.sample, rename it seed.yaml and fill in the value for the private key, using the key generated above or shared by a colleague.
 
 ### Update the WAF
 
-The WAF will block attempts made by the local application to use the build api unless some rules are switched off.  This can be done temporarily in the build environment during testing.  When a deployment happens the rules will be reset, but it's a good idea to keep the rules the same in all environments as much as possible, so these rules have not been switched off permanently in build.
+The WAF will block attempts made by the local application to use the build api unless some rules are switched off. This can be done temporarily in the build environment during testing. When a deployment happens the rules will be reset, but it's a good idea to keep the rules the same in all environments as much as possible, so these rules have not been switched off permanently in build.
```

Comment:
I'm still a bit nervous about this, but it seems to work so let's leave it for now and come back if we have any problems.

Reply:
Ah, I didn't set out to change this 🤔 I think this is my text linter getting rid of whitespace...

Reply:
Yeah so this came in as content here:
https://github.com/alphagov/di-authentication-account-management/commit/93a19ac35a5059f29a73e0d7557aa216d583f088

I'm guessing your nerves are about the content and not the whitespace linting @alex9smith

Reply:
Ah yeah, makes sense! Yup I'm concerned about the content but like I said it seems to work fine